### DISCUSSION TITLE
TP1 Ejercicios 8,9 y 11

### DISCUSSION THREAD

**Discussion by Lautymendez** [INGCOM-UNRN-P1/cursada-2023#63] [Preguntas y respuestas] [answered]
Buen dia Martin, quisiera saber si hay una posibilidad de que me ayudes a resolver estos ejercicios. Ya he armado de antemano el programa pero siguen sin funcionar, quiza hay un error de logica o algo que no estoy viendo. 

Gracias

**Comment by martinvilu** [comment]
Subilos al repositorio y cargá el enlace acá asi los reviso

**Reply by Lautymendez** [reply]
https://github.com/INGCOM-UNRN-P1/tp1-2023-Lautymendez

Ese es el repositorio privado que se crea (nose si esta bien el link)

**Reply by Lautymendez** [reply]
El ejercicio 9 no lo pude empezar porque no llegue a decifrar el algoritmo todavia, pero los ejercicios 8 y 11 si estan armados

**Reply by martinvilu** [reply]
Para el ejercicio 9, tenes que ver si el resto de la división da o no cero en la función

**Reply by martinvilu** [reply]
El repositorio esta vacio; te faltó hacer `git push`

**Reply by Lautymendez** [reply]
Que raro porque hice push en todos los ejercicios, y aparte estoy haciendo push nuevamente y me sale lo que adjunto en la foto

quiza esta mal el link
![error](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141681157/7e2d9315-86c8-4da0-a163-d5f48b2e44f3)

**Reply by martinvilu** [reply]
Por el nombre del directorio donde esta la terminal, no _parece_ ser el repositorio del TP; comparalo con el enlace que me pasaste.

**Reply by Lautymendez** [reply]
https://github.com/INGCOM-UNRN-P1/tp1-2023-Lautymendez/commits/master

Ahi encontre los commits recientes, ese link los podes ver?

**Reply by martinvilu** [reply]
🤦 y por que estas en otra rama... después vemos como pasar de `master` a `main` (como corresponde)

**Reply by Lautymendez** [reply]
Perdon hace poco estoy aprendiendo git, mala mia

**Reply by martinvilu** [reply, marked as answer]
Nada grave, despues lo vemos como arreglar

**Reply by Lautymendez** [reply]
Buenisimo, cuando pueda confirmeme si ha podido revisar los ejercicios que le mencione, asi entrego el tp hoy porfas

**Reply by martinvilu** [reply]
Para el proximo TP; hacer `git clone https://...` sobre el repositorio que se crea. **Sin** crear un repositorio nuevo en tu computadora primero.

Asi no vuelve a suceder esto.

**Reply by martinvilu** [reply]
Voy a arreglar esto, pero necesito que vuelvas a hacer clone sobre el repositorio; lee el apunte de git también, ahí están los pasos a seguir para el trabajo normal.

Si el TP2 esta de la misma manera, no lo voy a arreglar y si no aparece en el pull request, no puede ser corregido.

**Reply by Lautymendez** [reply]
Dale Martin no te preocupes, lo voy hacer para el proximo TP y tambien para el tp 1 que estoy terminando. Gracias!

**Comment by martinvilu** [comment]
**MUY IMPORTANTE** para seguir trabajando, hace `git clone` denuevo, y sin un repositorio creado. (denuevo, lee el apunte de git)

**Reply by martinvilu** [reply]
Lo veo, pero lo revisaré de la misma manera que el resto; cargalo en el formulario de entrega.

**Reply by Lautymendez** [reply]
Buenisimo ahora me pongo a corregir y cuando este listo envio el trabajo. Muchas gracias por su tiempo!

**Reply by Lautymendez** [reply]
Dale, pongo git clone "(link del tp1)"  ?

**Comment by Lautymendez** [comment]
> **MUY IMPORTANTE** para seguir trabajando, hace `git clone` denuevo, y sin un repositorio creado. (denuevo, lee el apunte de git)
